conftest.py

```diff
@@ -73,9 +73,9 @@ def environ():
 
 
 @pytest.fixture
-def experiment(tmp_path, environ):  # pylint: disable=redefined-outer-name,unused-argument
+def experiment(environ):  # pylint: disable=redefined-outer-name,unused-argument
     """Mock an experiment."""
-    os.environ['WORK'] = str(tmp_path)
+    os.environ['WORK'] = '/directory-not-on-host'
```

Comment:
Does this work ?

Reply:
Not unless the user of this fixture makes it in fakefs, or reassigns it to something else.

Reply:
This is just confusing, lets just use /work, it wont exist on host, but a little clear when debugging, since /directory-not-on-host is not clear when you get in output.

Reply:
Done